package.json

```diff
@@ -78,6 +78,7 @@
     "express": "^4.12.3",
     "filesize": "^3.1.3",
     "find-up": "^1.1.2",
+    "fixturify": "^0.3.0",
```

Comment:
Are we actually using this outside of tests?  Seems like this should be in `devDependencies`?